amazon_transcribe/serialize.py

```diff
@@ -59,8 +60,17 @@ def serialize(self) -> Tuple[Dict[str, HEADER_VALUE], BufferedIOBase]:
             "x-amzn-transcribe-media-encoding": self.request_shape.media_encoding,
             "x-amzn-transcribe-vocabulary-name": self.request_shape.vocabulary_name,
             "x-amzn-transcribe-session-id": self.request_shape.session_id,
+            "x-amzn-transcribe-vocabulary-filter-name": self.request_shape.vocab_filter_name,
             "x-amzn-transcribe-vocabulary-filter-method": self.request_shape.vocab_filter_method,
+            "x-amzn-transcribe-vocabulary-show-speaker-label": self.request_shape.show_speaker_label,
+            "x-amzn-transcribe-enable-channel-identification": self.request_shape.enable_channel_identification,
+            "x-amzn-transcribe-number_of_channels": self.request_shape.number_of_channels,
```

Comment:
Was this supposed to be underscores in the header name? It looks like the rest of them are using dashes.